.dccache

```diff
@@ -0,0 +1 @@
:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_invisible.py":[1482,1639462933989.0151,"675d0c01f372632aae476a53226b114b2cef0057a7e5f2b3e2be30831b7eb546"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_lookups.py":[1770,1639462933990.0127,"d680e6c4d3db84aa20c86d7a943d4c657505c6d8d782b73e26fa43898b2ec905"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_many2many.py":[2448,1639462933991.015,"a6b66b6eceeeab6eef7f4d50b28791c1d1d84f408e6454dbc14577ae6b36ef4a"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_no_cascade.py":[2219,1639462933991.015,"09667efe0bf5a28efd83562e5c1344add16bb3df613cd37e9d6aed18c4c6e9ba"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_no_delete.py":[1004,1639462933992.014,"247a7312c7ca87f7179efd5e8cd01bc03291e1d628828028b3bcfdf1542a30bb"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_prefetch_related.py":[1858,1639462933993.0166,"2bc4ce460f9b40aa2da36f87c7aa4d09649f6986d10593cf8b18fbaf8c518502"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_queryset.py":[9222,1639462933994.0164,"f8cdffb44e73687b9a6b451495a18cf3ff6acbf108e32a74d3302f87258945a1"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_refresh_from_db.py":[689,1639462933994.0164,"40c3c9221dca4726bd0c5520fd75f3f1c3a2febf1ab64f53e747c3f89ee768a2"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_single_delete_call_on_inheritance.py":[2081,1639462933995.0154,"649fe24c634f6f59b9bec446e7b8ad08788c6054c4646c93e1a8283d2f560ff8"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_soft_delete.py":[7538,1639462933995.0154,"48074a24d554fecda61acfef85ad8faad6cc46cc959568a98c2769e00ee3812f"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_soft_delete_cascade.py":[6337,1639462933996.0156,"e9cd0f1fa8cf02dce625ca4ce13e5ef24bed61d74c9ef2e6d720eaf1697071fc"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_subquery.py":[1393,1639462933997.0176,"32e55d828929f7d48bb1de0675cd79f7e06c358dd5a25818b1abe94f61e4d78f"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/test_visible_by_pk.py":[2824,1639462933997.0176,"a0613d389d31ece51433211577506d6505c526f21afbfdbccd22ba2ce2aba5d4"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/urls.py":[568,1639462933998.1387,"9deae4c1e3634ab7922bd5121bee20a4e3b6643ff04c368915164038aaefefe6"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/tests/__init__.py":[0,1639462933985.0159,"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"],"c:/Users/moshf/Desktop/Dev/Projects/django-safedelete/safedelete/templates/safedelete/undelete_selected_confirmation.html":[846,1639462933984.0146,"9d38b0e2d4e1d1ecd776b2a94b5579e0a23e253d7801bdb97811ba3fea28e78f"]}
```

Comment:
Don't commit that file (you can add it to .gitignore)

Reply:
done. Thanks!